.github/workflows/ci_linux.yml

```diff
@@ -47,6 +47,7 @@ jobs:
         sudo apt -y install libsdl2-dev libva-dev python3-venv valgrind spirv-tools glslang-dev libvulkan-dev mesa-vulkan-drivers vulkan-validationlayers
         sudo apt -y install ffmpeg libavcodec-dev libavutil-dev libavformat-dev libavdevice-dev libavfilter-dev libswscale-dev libswresample-dev libpostproc-dev
         sudo apt -y install libfreetype-dev libharfbuzz-dev libfribidi-dev
+        sudo apt -y install graphviz
```

Comment:
I think you can just drop `Nope` from the commit title `doc: add a custom snippet rendering plugin for Nope scenes`.